package.json

```diff
@@ -57,6 +57,10 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "@cypress/webpack-preprocessor": "^4.1.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.19",
+    "@fortawesome/free-regular-svg-icons": "^5.9.0",
+    "@fortawesome/free-solid-svg-icons": "^5.9.0",
+    "@fortawesome/react-fontawesome": "^0.1.4",
```

Comment:
The core FA library for JS / tree-shaking, the free icons and the React wrapper.
The original implementation used CSS pseudo-elements instead of the React wrappers but the table / FA core interaction for actively checking pseudo-elements degraded the table's performance significantly (~300% increase in render time)